src/main/pages/setup-kubernetes/kubernetes-multi-user.adoc

```diff
@@ -37,7 +37,7 @@ kubectl create clusterrolebinding add-on-cluster-admin --clusterrole=cluster-adm
 kubectl config use-context <my-context>
 ----
 +
-* Install `tiller`, which is the Helm server, on your cluster.
+* Install the Helm server `tiller` on your cluster.
```

Comment:
This doesn't make sense. I wrote to leave out "which is", therefore:

Install `tiller`, the Helm server, on your cluster.

Reply:
Done.